### PR TITLE
fix signup logging regression, refs #331

### LINKED ISSUE #331: log new signup actions
previously we'd log `signup_start` only if the visitor landed on the page & the step is `username` -- this isn't the case anymore though, since #327 

see https://github.com/steemit/faucet/blob/master/src/components/Signup.js#L96

we should track clicks on either of the two free/paid signup buttons, and also actions taken after that (outbound link clicks, closing modal & clicking free option button)

### PR DESCRIPTION
**Pull request by gl2748**
closes #331

### CODE REVIEW

**Comment by roadscape** [comment]
Lgtm, approving once step names clarified

File: src/components/Form/Signup/SignupOptions.js [hunk 1]
```diff
@@ -82,7 +86,13 @@ const SignupOptions = ({
                     <FormattedMessage id="signup_options_blocktrades" />
                 </p>
 
-                <a className="external-link" href="https://anon.steem.network/">
+                <a
+                    className="external-link"
+                    href="https://anon.steem.network/"
+                    onClick={() => {
+                        logCheckpoint('anonSteem_signup_chosen');
```
Comment (bnchdrff) [inline src/components/Form/Signup/SignupOptions.js]: no need to throw in the camelcase here!

**Comment by gl2748** [comment]
Makes sense, will do.
for comepleteness I now have:
```
free_signup_chosen
paid_signup_clicked_anonsteem
paid_signup_clicked_blocktrades
paid_signup_clicked_steemconnect
```

**Comment by gl2748** [comment]
@bnchdrff changes ok?

**Pull request merged**